src/v2/layouts/Files/FileList/FileList.jsx

```diff
@@ -0,0 +1,106 @@
+import Accordion from '../../../components/accordion/Accordion';
+import { Icon } from '../../../components/design/Icon/Icon';
+import { faDownload, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import { useMutation } from '@tanstack/react-query';
+import { API } from '../../../components/providers';
+import { deleteDocument } from '../filesApi';
+import { useState } from 'react';
+import { DeleteConfirmationModal } from './DeleteConfirmationModal';
+import { formatFileList } from './FileListUtils';
+
+export const FileList = ({ fetchDocumentsResp, fetchDocument, onClickViewFile }) => {
+	const [showDeleteConfirmPopup, setShowDeleteConfirmPopup] = useState(false);
+	const [deleteFileId, setDeleteFileId] = useState();
+	const { mutate: deleteDocuments } = useMutation({
+		mutationKey: [API.DELETE_DOCUMENT],
+		mutationFn: deleteDocument,
+		onSuccess: () => {
+			fetchDocument();
+			onCloseModal();
+		},
+	});
+
+	const onCloseModal = () => {
+		setDeleteFileId(null);
+		setShowDeleteConfirmPopup(false);
+	};
+
+	const onDeleteFile = () => {
+		deleteDocuments({
+			fileId: deleteFileId,
+		});
+	};
+	return (
+		<>
+			{showDeleteConfirmPopup && (
+				<DeleteConfirmationModal onCloseModal={onCloseModal} onDeleteFile={onDeleteFile} />
+			)}
+			<StyledAccordion>
+				{fetchDocumentsResp &&
+					formatFileList(fetchDocumentsResp)?.map((documentsObj) => {
+						return (
+							<Accordion key="document" title={documentsObj.title}>
```

Comment:
key should not be same for all the items, 
```suggestion
							<Accordion key={documentsObj.id} title={documentsObj.title}>
```